shared/middleware/helpers/storage_apps.rb

```diff
@@ -14,7 +14,7 @@ def initialize(storage_id)
   def create(value, ip:, type: nil, published_at: nil)
     timestamp = DateTime.now
     row = {
-      storage_id: @storage_id,
+      storage_id: @storage_id.to_i,
```

Comment:
Without this `to_i`, I'm seeing the following error (stack trace).
```
Sequel::DatabaseError:         Sequel::DatabaseError: Mysql2::Error: Incorrect integer value: 'de96e1f00e331d46db5235f93d5dadbf' for column 'storage_id' at row 1
            app/models/channel_token.rb:65:in `create_channel'
            app/models/channel_token.rb:36:in `block (2 levels) in find_or_create_channel_token'
            app/models/channel_token.rb:34:in `block in find_or_create_channel_token'
            app/models/channel_token.rb:32:in `find_or_create_channel_token'
            test/models/channel_token_test.rb:14:in `block in <class:ChannelTokenTest>'
            test/testing/setup_all_and_teardown_all.rb:22:in `run'
```
Is this a mistake in my code, test data, or otherwise?